ogr/services/gitlab/pull_request.py

```diff
@@ -106,6 +107,17 @@ def diff_url(self) -> str:
     def commits_url(self) -> str:
         return f"{self._raw_pr.web_url}/commits"
 
+    @property
+    def patch(self) -> bytes:
+        response = requests.get(f"{self._raw_pr.web_url}.patch")
```

Comment:
I see we already have a `url` property you can use:
```suggestion
        response = requests.get(f"{self.url}.patch")
```